### ISSUE TITLE
polyserve failed on cygwin, win7

### ISSUE THREAD

**Issue by renfeng** [PolymerLabs/polyserve#42]
frren@FRREN-CN ~/git/codelab-components
$ polyserve
C:\Users\frren\AppData\Roaming\npm\node_modules\polyserve\bin\polyserve:14
const resolve = require('resolve');
^^^^^
SyntaxError: Use of const in strict mode.
at exports.runInThisContext (vm.js:73:16)
at Module._compile (module.js:443:25)
at Object.Module._extensions..js (module.js:478:10)
at Module.load (module.js:355:32)
at Function.Module._load (module.js:310:12)
at Function.Module.runMain (module.js:501:10)
at startup (node.js:129:16)
at node.js:814:3

**Comment by MeTaNoV** [comment]
Which version of node are you using?

**Comment by renfeng** [comment]
should be the latest... will check later

**Comment by hickscorp** [comment]
Hi,

I just installed Polyserve on a Linux box, and i am getting the same error:

```
const resolve = require('resolve');
^^^^^
SyntaxError: Use of const in strict mode.
    at exports.runInThisContext (vm.js:73:16)
    at Module._compile (module.js:443:25)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:129:16)
    at node.js:814:3
```

Here is the output of `npm info polyserve`:
```
{ name: 'polyserve',
  description: 'A simple dev server for bower components',
  'dist-tags': { latest: '0.5.0' },
  versions: [ '0.1.0', '0.2.0', '0.2.1', '0.3.0', '0.4.0', '0.5.0' ],
  maintainers: 'justinfagnani <justin@fagnani.com>',
  time: 
   { modified: '2015-11-10T21:26:27.405Z',
     created: '2015-03-19T00:04:27.156Z',
     '0.1.0': '2015-03-19T00:04:27.156Z',
     '0.2.0': '2015-03-19T18:44:23.031Z',
     '0.2.1': '2015-03-19T21:05:55.033Z',
     '0.3.0': '2015-04-03T18:02:35.145Z',
     '0.4.0': '2015-05-01T18:14:41.903Z',
     '0.5.0': '2015-11-10T21:26:27.405Z' },
  license: 'BSD-3-Clause',
  readmeFilename: 'README.md',
  users: { tyriar: true, hongkheng: true },
  homepage: 'https://github.com/polymerlabs/polyserve#readme',
  keywords: [ 'polymer', 'polyserve' ],
  repository: 
   { type: 'git',
     url: 'git+https://github.com/polymerlabs/polyserve.git' },
  bugs: { url: 'https://github.com/polymerlabs/polyserve/issues' },
  version: '0.5.0',
  main: 'main.js',
  bin: { polyserve: 'bin/polyserve' },
  scripts: { start: './bin/polyserve' },
  dependencies: 
   { express: '^4.8.5',
     'find-port': '^1.0.1',
     minimist: '^1.1.1',
     opn: '^3.0.2',
     resolve: '^1.0.0',
     send: '^0.10.1' },
  gitHead: '59561582d963f9d32045b1837f9b8b0e5f3787fd',
  dist: 
   { shasum: '97510dd83a8020a127c27bd0f11938ad14e113aa',
     tarball: 'http://registry.npmjs.org/polyserve/-/polyserve-0.5.0.tgz' },
  directories: {} }
```

Not sure how to find out what exact version I'm using (Issuing `polyserve --version` of course triggers the same error), any hint please?

**Comment by turbidwater** [comment]
I'm having the same issue with node v0.10.31 on Win10

**Comment by MeTaNoV** [comment]
you need at least 0.12.x, the later, the better!

**Comment by turbidwater** [comment]
@MeTaNoV I updated to 0.12.7 and had the same result. I then made the leap to 5.1.0 and it does work there.

**Comment by MeTaNoV** [comment]
perfect! then, it is v4.x minimum...

**Comment by mbleigh** [comment]
+1 to this issue, Node 0.12.7 on Ubuntu.

Would be great if this worked on pre-Node-4.0 versions.

**Comment by nchathu2014** [comment]
Guys You have to update it to latest.At this time it is 5.3.0v and the same error is resolved for me when I got updated to latest.

**Comment by rictic** [comment]
Related to #50

**Comment by justinfagnani** [comment]
Should be fixed by #51. Please reopen if not.
